Userland/Utilities/shot.cpp

```diff
@@ -38,17 +38,26 @@ int main(int argc, char** argv)
     Core::ArgsParser args_parser;
 
     String output_path;
+    int file_append = 0;
     bool output_to_clipboard = false;
 
     args_parser.add_positional_argument(output_path, "Output filename", "output", Core::ArgsParser::Required::No);
     args_parser.add_option(output_to_clipboard, "Output to clipboard", "clipboard", 'c');
 
     args_parser.parse(argc, argv);
 
-    if (output_path.is_empty()) {
+    if (output_path.is_empty())
         output_path = Core::DateTime::now().to_string("screenshot-%Y-%m-%d-%H-%M-%S.png");
+    
+    while (Core::File::exists(output_path)) {
+        file_append += 1;
+        if (!Core::File::exists(String::formatted("{}({}).png", output_path.split('.')[0], file_append)))
+            break;
     }
```

Comment:
This is okay for auto-generated file names but if I'm specifying a name, I don't want the program to mess with it.

Reply:
@awesomekling exactly the opposite, auto-generated names is the date in which the file was created, but if you specify a name let's say `shot file.png`, and then you do it again `shot file.png`, with my patch the second file will be renamed to `file(1).png` to not accidentally overwrite the first screenshot.

Reply:
If the user runs `shot x.png`, we should respect that and create `x.png`. Changing the name automatically would violate expectations and does not match the behavior of any other program.